labman/gui/templates/plate.html

```diff
@@ -131,18 +131,12 @@ <h4>Plate map</h4>
   <p>Press <kbd class='key'>ESC</kbd> to enter cell <i>selection mode</i></p>
   <p>Press <kbd class='key'>ENTER</kbd> or double click a cell to enter <i>edit mode</i></p>
     <table>
-      <tr>
-        <td>blank:</td>
-        <td>gDNA extraction blanks. Represents an empty extraction well.</td>
-      </tr>
-      <tr>
-        <td>vibrio.positive.control: </td>
-        <td> Bacterial isolate control (Vibrio fischeri ES114). Represents an extraction well loaded with Vibrio.</td>
-      </tr>
-      <tr>
-        <td>zymo.mock:</td>
-        <td>Bacterial community control (Zymo Mock D6306). Represents an extraction well loaded with Zymo Mock community.</td>
-      </tr>
+      {% for cd in controls_description %}
```

Comment:
I would add a note at the foot of this table along the lines of: "To add new blank/sample types go to Admin>Manage control sample types" (perhaps even link to it).

Reply:
Done